easybuild/framework/easyblock.py

```diff
@@ -2984,9 +2985,16 @@ def sanity_check_rpath(self, rpath_dirs=None):
         self.log.debug("$LD_LIBRARY_PATH during RPATH sanity check: %s", os.getenv('LD_LIBRARY_PATH', '(empty)'))
         self.log.debug("List of loaded modules: %s", self.modules_tool.list())
 
-        not_found_regex = re.compile('not found', re.M)
+        not_found_regex = re.compile(r'([^\s]*) \=\> not found')
```

Comment:
> Ah, no, it's because of the switch from search to findall below, OK.

Yes, indeed

> But my suggestion on [^\s]* vs \S+ remains.

Valid point, \S+ does the job, I'll replace it